packages/expo/cli/utils/api.ts

```diff
@@ -0,0 +1,75 @@
+import { JSONValue } from '@expo/json-file';
+import got, { HTTPError, NormalizedOptions, RequestError } from 'got';
```

Comment:
My understanding was that we agreed not to use axios or got so we'll need to circle back and replace this with node-fetch.